packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
```

Comment:
It seems like that would be in keeping with the other changes to defer to `sw-build`'s defaults, so 👍